.env.example

```diff
@@ -6,3 +6,4 @@ MONGO_PORT=27017
 MONGO_DATABASE=testnet-statistics
 SLACK_WEBHOOK_SECRET_URL=https://hooks.slack.com/services/abc/def/123456
 FF_NOTIFY_HEALTH_SLACK=false
+STATISTICS_EXCLUDED_TO_ADDRESSES=0x559dBa9F569846F2bd7181F8aeE7CF3D72cB43C6,0xd3b7c0c75aF24aceB19b4D6d4c144520E4f4B3E1,0x34f383ca78AdDC253C49D913CaED21A324C1aa33
```

Comment:
suggestion(non-blocking): I prefer something like a js config file for settings like this, as this can get quite long and confusing if we have to add new addresses in the future.
```ts
{
  "config": {
    "contractBlacklist": [
      "0x...",
      "0x...",
      "0x...",
      "0x..."
    ]
  }
}
```

Reply:
Would be okay to merge it like is for now and open an issue to consider this in the future. Should probably ref in a new issue.